tests/purs/failing/NestedRecordLabelOnTypeError.out

```diff
@@ -0,0 +1,28 @@
+Error found:
+in module [33mNestedRecordLabelOnTypeError[0m
+at tests/purs/failing/NestedRecordLabelOnTypeError.purs:5:10 - 5:22 (line 5, column 10 - line 5, column 22)
+
+  Could not match type
+  [33m    [0m
+  [33m  ()[0m
+  [33m    [0m
+  with type
+  [33m                [0m
+  [33m  ( field :: Int[0m
+  [33m  | t0          [0m
+  [33m  )             [0m
+  [33m                [0m
+
+while checking that expression [33m{ field: 1[0m
+                               [33m}         [0m
+  has type [33m{ a :: { b :: { c :: Int[0m
+           [33m              }         [0m
+           [33m       }                [0m
+           [33m}                       [0m
+in value declaration [33mrecord[0m
+
+where [33mt0[0m is an unknown type
```

Comment:
Looks like there is no mention of the label here as is expected based on your comment in the corresponding `purs` file.